.github/workflows/pre-commit.yaml

```diff
@@ -0,0 +1,56 @@
+name: Common issues check
+
+on: [pull_request]
+
+jobs:
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+    - run: |
+        git fetch --no-tags --prune --depth=1 origin +refs/heads/*:refs/remotes/origin/*
+
+    - name: Get changed files
+      id: file_changes
+      run: |
+        export DIFF=$(git diff --name-only origin/${{ github.base_ref }} ${{ github.sha }})
+        echo "Diff between ${{ github.base_ref }} and ${{ github.sha }}"
+        echo "files=$( echo "$DIFF" | xargs echo )" >> $GITHUB_OUTPUT
+
+    - name: Install shfmt
+      run: |
+        curl -L "$(curl -s https://api.github.com/repos/mvdan/sh/releases/latest | grep -o -E -m 1 "https://.+?linux_amd64")" > shfmt \
+        && chmod +x shfmt && sudo mv shfmt /usr/bin/
+
+    - name: Install shellcheck
+      run: |
+        sudo apt update && sudo apt install shellcheck
+
+    - name: Install hadolint
```

Comment:
Not sure if it's used (?), couldn't find reference anywhere.

Reply:
Nope, removed